src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
```

Comment:
It appears that this class is meant to represent a transaction, but it lacks some crucial transactional attributes, such as a timestamp or an identifier. Consider adding these attributes to fully represent a transaction.